cedar-lean/Cli/Main.lean

```diff
@@ -30,7 +30,7 @@ def readFile (filename : String) : IO String :=
 def printUsage (err : String) : IO Unit :=
   IO.println s!"{err}\nUsage: Cli <command> <file>"
 
-def main (args : List String) : IO Unit :=
+unsafe def main (args : List String) : IO Unit :=
```

Comment:
Just out of curiousity, why is this needed now?

Reply:
It's the result of calling `unsafeBaseIO` in `Cli/Main.lean` in order to pull out the timing info from an IO monad. A little hacky -- open to better suggestions 🙂